src/org/scratchOrgInfoApi.ts

```diff
@@ -7,8 +7,7 @@
 
 import { env, Duration, upperFirst } from '@salesforce/kit';
 import { AnyJson } from '@salesforce/ts-types';
-// @ts-ignore
-import { OAuth2Config, OAuth2Options, SaveResult } from 'jsforce';
+import { OAuth2Config, JwtOAuth2Config, SaveResult } from 'jsforce';
```

Comment:
these changes are worth a look.  jsforce2/JwtOAuth2Config (extends its OAuth2Config ) includes all the fields we needed.

the previous type had been ts-ignored because **it didn't exist**